Chapter1_Challenge3/Menu.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Chapter1_Challenge3
+{
+    public static class Menu
+    {
+        public static void ShowMenu()
+        {
+            Console.WriteLine("    Menu   ");
+            Console.WriteLine();
+            Console.WriteLine("1) Register");
+            Console.WriteLine("2) Login"); 
+            Console.WriteLine("3) Exit programm");
+            Console.WriteLine();
+            Console.Write("Your Choice: "); 
+        }
+
+        public static int InputAndValidateChoice()
+        {
+           var valid = int.TryParse(Console.ReadLine(), out int choice);
+
+            if (!valid && choice < 1 || choice >3)
```

Comment:
Still as bad. With constants it might be better.

Reply:
Must I then make the class not static because otherwise I see this error message 

Menu.LowestMenuChoice': cannot declare instance members in a static class.

or is there another way to make this work